dll/win32/imm32/compstr.c

```diff
@@ -6,7 +6,7 @@
  *              Copyright 2002, 2003, 2007 CodeWeavers, Aric Stewart
  *              Copyright 2017 James Tabor <james.tabor@reactos.org>
  *              Copyright 2018 Amine Khaldi <amine.khaldi@reactos.org>
- *              Copyright 2020 Oleg Dubinskiy <oleg.dubinskij2013@yandex.ua>
+ *              Copyright 2020 Oleg Dubinskiy <oleg.dubinskij30@gmail.com>
```

Comment:
Maybe it is enough to have 3 maintainers listed here, instead of 4. Just a thought. To reduce future grooming efforts.

Reply:
I would recommend then to strip instead "your line" in all files where we do have 3+. This will not be the last time your email might change, and you also will not the only guy with this issue. Let's improve today, when we do touch the lines anyway.

Reply:
OK, done @JoachimHenze. :wink: